JavaScripit/index.js

```diff
@@ -70,3 +70,38 @@ messageForm.addEventListener('submit', function (event) {
     messageForm.reset();
 });
 
+var githubRequest = new XMLHttpRequest();
+var githubUsername = "Haregebezu";
+var url = "https://api.github.com/users/" + githubUsername + "/repos";
```

Comment:
```suggestion
const url = `https://api.github.com/users/${githubUsername}/repos`;
```
Good place for template.